packages/ERTP/src/issuer.chainmail

```diff
@@ -0,0 +1,209 @@
+/**
+ * The issuer cannot mint new amount, but it can create empty purses and
+ * payments. The issuer can also transform payments (splitting payments,
+ * combining payments, burning payments, and claiming payments
+ * exclusively). The issuer should be gotten from a trusted source and
+ * then relied upon as the decider of whether an untrusted payment is valid. 
+ */
+interface Issuer (Amount (Extent)) {
+  /**
+   * Get the Brand for this Issuer. The Brand indicates the kind of
+   * digital asset and is shared by the mint, the issuer, and any purses
+   * and payments of this particular kind. The brand is not closely
+   * held, so this function should not be trusted to identify an issuer
+   * alone. Fake digital assets and amount can use another issuer's brand.
+   */
+  getBrand() -> (Brand);
+
+  /* Get the allegedName for this mint/issuer */
+  allegedName() -> (allegedName);
+
+  /* Get the AmountMath for this Issuer. */
+  getAmountMath() -> (AmountMath);
+
+  /* Get the name of the MathHelpers for this Issuer. */
+  getMathHelpersName() -> (String);
+
+  /** Make an empty purse of this brand. */
+  makeEmptyPurse() -> (Purse);
+
+  /**
+   * Get payment balance. Because the payment is not trusted, we
+   * cannot call a method on it directly, and must use the issuer
+   * instead.
+   */ 
+  getBalance(payment :Payment) -> (Amount);
+
+  /** 
+   * Burn all of the digital assets in the payment. `Amount` is optional.
+   * If `amount` is present, the code will insist that the payment balance is equal
+   * to `amount`, to prevent sending the wrong payment and other confusion.
+   */
+  burn(payment :Payment, amount :Amount) -> (Amount);
```

Comment:
Obviously, all the comments about parameter naming in issuer.js apply here as well. Unless there's a good reason not to, the parameter lists should be the same.

Reply:
Yup, just not updated yet.